banners/english/C24_WMDE_Desktop_EN_02b/styles/styles.scss

```diff
@@ -0,0 +1,59 @@
+// This is the file where we import the theme-specific component styles
+@use 'src/themes/Treedip/swatches/skin_vector-2022' with (
+	$slider: true,
+	$select-group-bubble: true,
+	$select-group-image-label: true,
+	$upgrade-to-yearly: true,
+	$fallback-banner: true,
+	$progress-bar: true,
+	$soft-close: true,
+);
+@use 'src/themes/Treedip/variables/breakpoints';
+@use 'src/components/BannerConductor/banner-transition';
+@use 'Banner';
+@use 'MainBanner' with (
+	$banner-height: 355px,
+	$form-width: 300px
+);
+@use 'src/themes/UseOfFunds/swatches/skin_vector-2022' as uof-vector-2022;
+@use 'src/themes/UseOfFunds/UseOfFunds';
+@use 'src/themes/Treedip/defaults';
+@use 'src/themes/Treedip/ButtonClose/ButtonClose' with (
+	$padding: 4px,
+	$top: 12px,
+	$right: 8px,
+	$margin: -4.5px
+);
+@use 'src/themes/Treedip/ProgressBar/ProgressBarAlternative' with (
+	$progress-bar-margin: 0 15px
+);
+@use 'src/themes/Treedip/DonationForm/DonationForm';
+@use 'src/themes/Treedip/DonationForm/MultiStepDonation';
+@use 'src/themes/Treedip/DonationForm/BubbleForm/SelectGroupBubble';
+@use 'src/themes/Treedip/DonationForm/BubbleForm/SelectGroupRadioBubbles' with (
+	$height: 34px
+);
+@use 'src/themes/Treedip/DonationForm/BubbleForm/SelectCustomAmountRadioBubble' with (
+	$height: 34px
+);
+@use 'src/themes/Treedip/DonationForm/BubbleForm/SelectGroupBubbleImageLabel';
+@use 'src/themes/Treedip/DonationForm/BubbleForm/SelectGroupBubbleImageLabel_dark';
```

Comment:
You can delete this import, the swatching improvements handle the payment icon colours differently now. If you want you can also delete the file and the imports from the older EN desktop banners